src/main/java/io/micronaut/build/MicronautGradleEnterprisePlugin.java

```diff
@@ -82,20 +80,14 @@ private void configureGradleEnterprise(Settings settings,
                 }
                 buildCache.getLocal().setEnabled(localEnabled);
                 if (remoteEnabled) {
-                    buildCache.remote(HttpBuildCache.class, remote -> {
-                        remote.setUrl("https://ge.micronaut.io/cache/");
+                    buildCache.remote(ge.getBuildCache(), remote -> {
                         remote.setEnabled(true);
                         if (push) {
-                            String username = envOrSystemProperty(providers, "GRADLE_ENTERPRISE_CACHE_USERNAME", "gradleEnterpriseCacheUsername", "");
-                            String password = envOrSystemProperty(providers, "GRADLE_ENTERPRISE_CACHE_PASSWORD", "gradleEnterpriseCachePassword", "");
-                            if (!username.isEmpty() && !password.isEmpty()) {
+                            String accessKey = providers.systemProperty("GRADLE_ENTERPRISE_ACCESS_KEY").getOrNull();
+                            if (accessKey != null && !accessKey.isEmpty()) {
                                 remote.setPush(true);
```

Comment:
What about this?
```suggestion
                                remote.setPush(accessKey != null && !accessKey.isEmpty());
```

Reply:
Yes, I thought about doing that, however if we leave the `if` in place...I don't see a reason for that, and without the `if`, we can't report it to the user.

Reply:
Ah i see what you mean. Looks good then!